### DISCUSSION TITLE
Mac上能用vnpy_binance嘛？

### DISCUSSION THREAD

**Discussion by CYorick** [veighna-global/vnpy_evo#20] [Q&A]
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/117757861/210197393-f3c11c13-462b-45ee-a6f2-380db50e0a43.png">

python脚本启动报错，

**Comment by xldistance** [comment]
自己在vnpy.trader.constant.py里面加上
```
class Exchange(Enum):
    BINANCE = "BINANCE"
```

**Comment by xldistance** [comment]
这是加上交易所的枚举名称
